### PR TITLE
Add tests for admin custom emojis controller

### PR DESCRIPTION
**Pull request by shuheiktgw**
I added tests for admin/custom_emojis_controller and fixed small bug I found in the controller!
Thanks! 👍

### CODE REVIEW

File: spec/controllers/admin/custom_emojis_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    let(:image) { fixture_file_upload(Rails.root.join('spec', 'fixtures', 'files', 'emojo.png'), 'image/png') }
+
+    before do
+      put :update, params: { id: custom_emoji.id, custom_emoji: params }
+    end
+
+    context 'when parameter is valid' do
+      let(:params) { { shortcode: :updated, image: image } }
+
+      it 'succeeds in updating custom emoji' do
```
Comment (ThibG) [inline spec/controllers/admin/custom_emojis_controller_spec.rb]: This should work, but I can't find anything making use of that functionality (changing the shortcode), so I'm not sure that's intended?

**Review by ClearlyClaire — DISMISSED** [review]
Apart from the use of symbols rather than strings for shortcodes, and the other inline remark I have made, this looks good to me.

File: spec/controllers/admin/custom_emojis_controller_spec.rb (hunk 1 above)
Comment (shuheiktgw) [reply spec/controllers/admin/custom_emojis_controller_spec.rb]: The original `shortcode` is `test` (set it [here](https://github.com/shuheiktgw/mastodon/blob/7808f82d22f76c05efbc8169eae5826deab671bc/spec/controllers/admin/custom_emojis_controller_spec.rb#L57)) and the test asserts that the shortcode is updated to `updated`! 👍

**Comment by shuheiktgw** [comment]
@ThibG Thank you for your reviews! I fixed them! 👍

File: app/controllers/admin/custom_emojis_controller.rb [hunk 2]
```diff
@@ -56,7 +56,7 @@ def copy
       emoji.image = @custom_emoji.image
 
       if emoji.save
-        log_action :create, emoji
+        log_action :copy, emoji
```
Comment (Gargron) [inline app/controllers/admin/custom_emojis_controller.rb]: We [do not have translations for "copy"](https://github.com/tootsuite/mastodon/blob/master/config/locales/en.yml#L148-L176). Copying a custom emoji is essentially creating one. Please revert this

**Review by Gargron — CHANGES_REQUESTED** [review]

**Review by Gargron — APPROVED** [review]

**Pull request merged**